ArduCopter/GCS_Mavlink.cpp

```diff
@@ -508,6 +513,7 @@ static const ap_message STREAM_RC_CHANNELS_msgs[] = {
     MSG_SERVO_OUTPUT_RAW,
     MSG_RC_CHANNELS,
     MSG_RC_CHANNELS_RAW, // only sent on a mavlink1 connection
+    MSG_MANUAL_CONTROL,
```

Comment:
There's too many things in this stream. I'm not sure if we should be sending this by default, and leave it up to the stream rate request handling.